sys-devel/gcc/gcc-8.3.0_2019_05_24.recipe

```diff
@@ -326,7 +327,8 @@ INSTALL()
 		$gccLibDir/libstdc++.so.$libstdcxxSoVersion \
 		$gccLibDir/libstdc++.so.$libstdcxxLibVersion \
 		$libDir/
-	rm $gccLibDir/libstdc++*.a
+	rm $gccLibDir/libstdc++.a
+	cp $gccLibDir/libstdc++fs.a $developLibDir
```

Comment:
Is there no .so variant of this library?

Reply:
I didn't find, but I admit didn't check the Makefile.

Reply:
I checked the Makefile, as I see make only static stdc++fs.

Reply:
from https://gitlab.kitware.com/cmake/cmake/issues/17834
````
> Is libstdc++fs going to be a permanent feature, required to be linked for
> C++17 std:filesystem support, or will it be folded into the main libstdc++
> at some point?  If so, is it known when that point will be?

It will be part of libstdc++.so at some point, but we don't want to
export new symbols from the DSO until our C++17 support is complete
and stable, because once we export a symbol from the DSO we have to
keep it forever.

I expect it will be done for GCC 9.1